data/strings/strings.txt

```diff
@@ -13330,7 +13332,7 @@
 
   [login_to_make_edits_visible]
     tags = android,ios
-    en = Log in to OpenStreetMap.org so other users can see the changes you have made.
+    en = Login to upload your changes for other users.
```

Comment:
This is really about pushing changes to the global database, not "other users" which might be interpreted as OM.

So it should be

"Log in to openstreetmap.org to publish your changes to the world."

(Log in vs login is unclear to me.   I view login as computer jargon and log in as standard English.

Reply:
good points, i'll change the wording :)